examples/energy-monitor/ui/mobile_main.slint

```diff
@@ -12,7 +12,9 @@ import { MobileHeader } from "blocks/blocks.slint";
 import { MenuBackground } from "components/menu_background.slint";
 
 
-export component MobileMain {
+export component MobileMain inherits Window {
```

Comment:
MobileMain is used by MainWindow in desktop_window and that's still the entry point, even on mobile.
Window as non-top-level is a bad idea.

Reply:
Indeed, I somehow overlooked that. That's complete nonsense.

I was using MobileMain as entry point for the screen on esp32p4-ev board, because desktop_window.slint is just way to big and the mcu one is too small for the big screen. But the answer then is to have a dedicated entry point, not this change.

Apologies :)